File: backend/src/main/java/com/campus/Campus_Connect/features/honor/dto/response/MedalCandidateResponse.java
package com.campus.Campus_Connect.features.honor.dto.response;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MedalCandidateResponse {

    private Integer registrationId;

    private String name;

    private Integer courseId;

    private Integer admissionYear;

    private Boolean team;

    private String avatarUrl;
}

File: android-app/app/src/main/java/com/example/campusconnect/feature/events/data/remote/response/MedalCandidateResponse.kt
package com.example.campusconnect.feature.events.data.remote.response

import com.google.gson.annotations.SerializedName

data class MedalCandidateResponse(

    @SerializedName("registrationId")
    val registrationId: Int,

    @SerializedName("name")
    val name: String,

    @SerializedName("courseId")
    val courseId: Int?,

    @SerializedName("admissionYear")
    val admissionYear: Int?,

    @SerializedName("team")
    val team: Boolean,

    @SerializedName("avatarUrl")
    val avatarUrl: String?
)